packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMessageComponentAsRichTextEditBox.tsx

```diff
@@ -200,6 +209,22 @@ export const ChatMessageComponentAsRichTextEditBox = (
     if (!submitEnabled) {
       return;
     }
+    /* @conditional-compile-remove(rich-text-editor-image-upload) */
+    if (inlineImagesWithProgress && inlineImagesWithProgress.length > 0) {
+      const contentWithUpdatedInlineImagesInfo = getContentWithUpdatedInlineImagesInfo(
+        textValue,
+        inlineImagesWithProgress
+      );
+      const messageTooLong = isMessageTooLong(contentWithUpdatedInlineImagesInfo.length);
+      // Set contentValueWithInlineImagesOverflow state to display the error bar
+      setContentValueWithInlineImagesOverflow(messageTooLong);
+      // The change from the setContentValueOverflow in the previous line will not kick in yet.
+      // We need to relay on the local value of messageTooLong to return early if the message is too long.
```

Comment:
```suggestion
      // We need to rely on the local value of messageTooLong to return early if the message is too long.
```
I think this was meant to be rely?